src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Protocol/DocumentMapping/RazorMapToDocumentEditsResponse.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Text.Json.Serialization;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.Razor.Protocol.DocumentMapping;
+
+internal sealed record class RazorMapToDocumentEditsResponse
+{
+    [JsonPropertyName("edits")]
+    public required TextChange[] Edits { get; init; }
```

Comment:
As above, either making this `TextEdit[]` or calling it `Changes` would be clearer